casl-requirements.yml

```diff
@@ -0,0 +1,8 @@
+
+# This is the Ansible Galaxy requirements file to pull in the correct roles
+# to support the operation of CASL provisioning/runs. 
+
+# From 'infra-ansible'
+- src: https://github.com/redhat-cop/infra-ansible
```

Comment:
Hmm ... maybe an environment issue on your localhost? Everything seems to be working fine as-is here:
```
$ cat requirements.yml 
- src: https://github.com/redhat-cop/infra-ansible

$ ansible-galaxy install -r requirements.yml -p roles
- extracting infra-ansible to roles/infra-ansible
- infra-ansible was installed successfully

$ ls -l roles/
total 4
drwxrwxr-x. 6 obedin obedin 4096 Jul  3 10:05 infra-ansible

$ ansible-galaxy --version
ansible-galaxy 2.3.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.13 (default, May 10 2017, 20:04:28) [GCC 6.3.1 20161221 (Red Hat 6.3.1-1)]
```